crates/curp/tests/it/server.rs

```diff
@@ -297,7 +297,7 @@ async fn shutdown_rpc_should_shutdown_the_cluster() {
     ));
 
     let collection = collection_task.await.unwrap();
-    sleep_secs(7).await; // wait for the cluster to shutdown
+    sleep_secs(1).await; // wait for the cluster to shutdown
```

Comment:
Why change this sleep duration to 1s?

Reply:
Because it's a random bug, I must run it several times to reproduce.  The 7 sec is too long. Do we need to sleep for such a long duration?

Reply:
Is there another way to tell whether group is finished or not?